guides/assets/stylesrc/_main.scss

```diff
@@ -439,7 +449,7 @@ body.guide {
           background-repeat: no-repeat;
           background-position: top center;
           display: inline-block;
-          font-family: 'Calibre', sans-serif;
+          font-family: 'Calibre', "Noto Sans Arabic", "Heebo", sans-serif;
```

Comment:
Another font declaration... not sure if `a` is covered by those already?

Reply:
Cleaned up in 91ddfcc0